README.md

```diff
@@ -30,3 +30,21 @@ See [the wiki](https://github.com/GoogleChrome/chrome-types/wiki) for more.
 Running the code requires Node 16+ as well as a working version of Python (3 is preferred, but 2.7+ should work) installed on your system.
 This has only been tested on Linux and macOS.
 Python is used to convert Chromium's internal IDL format to JSON.
+
+**Steps**
```

Comment:
Could we put this under a new `### As a dependency` header? Steps seems a bit weird since the rest of the text above is talking about how to build.